index.html

```diff
@@ -4,12 +4,77 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Calculator</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=League+Spartan:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <link href="css/index.css" rel="stylesheet" />
   </head>
-  <body>
-    <main>
-      <h1>HELLO,WORLD</h1>
-    </main>
+  <body class="theme-one">
+    <div class="body-container">
+      <div class="title-container">
+        <h2 class="title">calc</h2>
+        <div class="theme-container">
+          <p class="theme-text">THEME</p>
+          <div class="theme-feature">
+            <form>
```

Comment:
Please give the form a name attribute so it can be used to target the form element .